buildutil.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"go/build"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/tools/go/vcs"
+)
+
+// forEachRepository calls found for each repository it finds in all GOPATH workspaces.
+func forEachRepository(found func(Repo)) {
+	for _, workspace := range filepath.SplitList(build.Default.GOPATH) {
+		srcRoot := filepath.Join(workspace, "src")
+		if _, err := os.Stat(srcRoot); os.IsNotExist(err) {
+			continue
+		}
+		_ = filepath.Walk(srcRoot, func(path string, fi os.FileInfo, err error) error {
```

Comment:
Yes, I use it to signify that the error is intentionally ignored.

I'll revisit this code and perhaps handle the error instead of ignoring it. The decision to ignore error came from older code, so it might not be a good decision.